tools/cdk-integ-tools/lib/integ-helpers.ts

```diff
@@ -183,13 +183,32 @@ export const DEFAULT_SYNTH_OPTIONS = {
     "ssm:account=12345678:parameterName=/aws/service/ami-amazon-linux-latest/amzn-ami-hvm-x86_64-gp2:region=test-region": "ami-1234",
     "ssm:account=12345678:parameterName=/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2:region=test-region": "ami-1234",
     "ssm:account=12345678:parameterName=/aws/service/ecs/optimized-ami/amazon-linux/recommended:region=test-region": "{\"image_id\": \"ami-1234\"}",
-    "vpc-provider:account=12345678:filter.isDefault=true:region=test-region": {
+    "vpc-provider2:account=12345678:filter.isDefault=true:region=test-region:subnetGroupNameTag=undefined": {
```

Comment:
For whatever reason, this `undefined` ends up in here. I'd rather not have that, it will make it super hard to canonicalize these strings.

Reply:
Changed.